tests/utils/inputparams/testparams.go

```diff
@@ -53,6 +54,15 @@ func GetContainerNameWithTimeStamp(containerName string) string {
 	return containerName + "_container_" + strconv.FormatInt(time.Now().Unix(), 10)
 }
 
+func GetVolumeNameOfSize(size int) string {
+	const chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
Nit: I'm fine with this special volume name. Just a little bit concern about the consistency: as we have discussed before, all test cases should create volumes (and other resources) with name starting with the test case name as the prefix, such as "restart_test_volume_xxxxx". The intention is to make it easier to keep track of volumes that are not cleaned up properly after running. I'm just thinking should we still follow that or not. Personally I tend to discard that convention, as it doesn't make much sense to me. What do you think? @pshahzeb @shuklanirdesh82